autometa/main.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+COPYRIGHT
+Copyright 2020 Ian J. Miller, Evan R. Rees, Kyle Wolf, Siddharth Uppal,
+Shaurya Chanana, Izaak Miller, Jason C. Kwan
+
+This file is part of Autometa.
+
+Autometa is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Autometa is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with Autometa. If not, see <http://www.gnu.org/licenses/>.
+COPYRIGHT
+
+Main script to run Autometa
+"""
+
+
+import logging
+import os
+import sys
+
+import multiprocessing as mp
+
+from .config.user import AutometaUser
+
+
+logger = logging.getLogger('autometa')
+
+
+__version__ = "2.0.0"
```

Comment:
Make sure the versions match (see other comments)

Reply:
Also, perhaps should just reference `VERSION` so that you don’t have to alter the version in so many places each time.

Reply:
I'll remove this from main

Reply:
I've also templated all of the instances of version for meta.yaml from the one instance within `setup.py`. Should only have to change `VERSION` now and everything will be updated appropriately.